app/assets/javascripts/app/form-field-format.js

```diff
@@ -1,17 +1,27 @@
-import Formatter from 'formatter.js-pebble';
+import FieldKit from 'field-kit';
+import validateField from './validate-field';
```

Comment:
stylistic nit: i like to separate the third party modules from our own modules with a blank line